datadog_checks_base/datadog_checks/base/checks/base.py

```diff
@@ -216,6 +216,55 @@ def __init__(self, *args, **kwargs):
         logger = logging.getLogger('{}.{}'.format(__name__, self.name))
         self.log = CheckLoggingAdapter(logger, self)
 
+        exclude_metrics_filters = self.instance.get('exclude_metrics_filters', []) if instance else []
+
+        if not isinstance(exclude_metrics_filters, list):
+            raise ConfigurationError('Setting `exclude_metrics_filters` must be an array')
+
+        self.exclude_metrics_filters = set()
+        self.exclude_metrics_pattern = None
+        exclude_metrics_patterns = []
+        for i, entry in enumerate(exclude_metrics_filters, 1):
+            if not isinstance(entry, str):
+                raise ConfigurationError('Entry #{} of setting `exclude_metrics_filters` must be a string'.format(i))
+
+            escaped_entry = re.escape(entry)
+            if entry == escaped_entry:
```

Comment:
Let's only support regexes as in practice this will never occur because basically all metrics contain dots

Reply:
Sure, makes sense!